src/main/python/systemds/operator/nodes/source.py

```diff
@@ -135,7 +131,7 @@ class Source(OperationNode):
 
     def __init__(self, sds_context, path: str, name: str):
         super().__init__(sds_context,
-                         f'"{path}"', output_type=OutputType.IMPORT)
+                         f'"{path}"')
```

Comment:
I have not created attribute for the IMPORT type since I have not seen code where the flag is relevant